util/src/Pos/Util/Log/LoggerConfig.hs

```diff
@@ -274,6 +275,25 @@ defaultInteractiveConfiguration minSeverity =
     in
     LoggerConfig{..}
 
+-- | Handler to override another console 'LogHandler' and disable logging
+-- to console altogether.
+disabledConsoleConfiguration :: LoggerConfig
+disabledConsoleConfiguration = LoggerConfig
```

Comment:
this just adds a 'no-op'; I don't think this was the intention.

Reply:
Well, this will collapse with other `LoggerConfig` called "console". I thought that's how logging to console was identified. 
I am adapting this and simply filtering out `StdOutBE` and `StdErrBE` :+1: